README.md

```diff
@@ -102,40 +95,19 @@ The notification processing service queries the records that have been received
 
 If you need to setup your cronjob in Magento <a href="http://devdocs.magento.com/guides/v2.0/config-guide/cli/config-cli-subcommands-cron.html" target="_blank">this is described here</a>.
 
-## Verified payment methods
-
- * Card payment methods
-    * Card payment method (non-3DS, 3DS1 and 3DS2)
-    * One click payment methods (using Billing agreements or Vault)
-    * Recurring payment methods (using Billing agreements or Vault) 
-   
-
-* Local payment methods
-   * Afterpay
-   * Apple Pay
-   * Bancontact
-   * Banktransfer IBAN
-   * Doku
-   * Google Pay
-   * iDeal
-   * Klarna
-   * Oney
-   * Zip
-   * Multibanco
-   * Konbini
-   * Swish
-   * MBway
-   * Twint
-   * Paypal
-   * SEPA Direct Debit
-   * PIX
-   * Ratepay
-   * Oxxo
-   * Econtexts
-   * PayBright
-   * Blik
-
-_This is not an extensive list of all the supported payment methods but only the ones that have been already verified. In case you would like to see other payment methods in the list as well please contact us to verify those too._
+## Supported payment methods
+
+* Ideal
+* AmazonPay
+* ApplePay
+* BcmcMobile
+* Dotpay
+* Facilypay3x
+* Multibanco
+* Googlepay
+* Klarna
+* Paypal
+* Sepadirectdebit
```

Comment:
This list is not complete and it is going to be unmanageable with our new approach so we are better off removing this section and replacing it with a link to the public docs page